components/FishForm.js

```diff
@@ -32,49 +32,47 @@ class FishForm extends Component {
   render() {
     return (
       <View style={styles.container}>
+        <Text style={styles.label}>Species:</Text>
         <TextInput
           style={styles.input}
           placeholder="Enter Species Name"
           value={this.props.currentFishEntry.species}
           onChangeText={(text) => this.handleChange("species", text)}
         />
 
+        <Text style={styles.label}>Length of the Fish:</Text>
         <TextInput
           style={styles.input}
-          placeholder="Enter Image"
-          value={this.props.currentFishEntry.image}
-          onChangeText={(text) => this.handleChange("image", text)}
-        />
-
-        <TextInput
-          style={styles.input}
-          placeholder="Enter Length"
+          placeholder="Enter Length (in)"
           keyboardType="number-pad"
           value={this.props.currentFishEntry.length.toString()}
           onChangeText={(text) => this.handleChange("length", parseInt(text))}
         />
 
+        <Text style={styles.label}>Weight of the Fish:</Text>
         <TextInput
           style={styles.input}
-          placeholder="Enter Weight"
+          placeholder="Enter Weight (lb)"
```

Comment:
Thanks for adding the labels to these text inputs.  This will make it better for the user to know what input is what and is easier for the user to navigate their information of entries.

Reply:
Agreed! Thank you for bringing this up. It was a great suggestion.